Windows CI failing: micromamba issue

E.g.: https://github.com/lfortran/lfortran/actions/runs/11048843229/job/30693104865?pr=4912
``` 
  set MAMBA_INSTALL_LOCN=C:\\Users\runneradmin\micromamba
  call %MAMBA_INSTALL_LOCN%\Scripts\activate.bat
  call micromamba activate lf
  set LFORTRAN_CMAKE_GENERATOR=Ninja
  set WIN=1
  set MACOS=0
  set ENABLE_RUNTIME_STACKTRACE=no
  call "C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Auxiliary/Build/vcvars64.bat"
  xonsh ci\build_tmp.xsh
  shell: C:\Windows\system32\cmd.EXE /D /E:ON /V:OFF /S /C "CALL "{0}""
  env:
    MACOSX_DEPLOYMENT_TARGET: 14
    MAMBA_ROOT_PREFIX: C:\Users\runneradmin\micromamba
    MAMBA_EXE: C:\Users\runneradmin\micromamba-bin\micromamba.exe
    CONDARC: D:\a\_temp\setup-micromamba\.condarc
    LFORTRAN_CMAKE_GENERATOR: Ninja
    CMAKE_C_COMPILER_LAUNCHER: ccache
    CMAKE_CXX_COMPILER_LAUNCHER: ccache
    ENABLE_RUNTIME_STACKTRACE: yes
'!__mamba_name!' is not recognized as an internal or external command,
operable program or batch file.
critical libmamba Shell not initialized

'micromamba' is running as a subprocess and can't modify the parent shell.
Thus you must initialize your shell before using activate and deactivate.


To automatically initialize all future (cmd.exe) shells, run:
    $ micromamba shell init --shell cmd.exe --root-prefix=~/.local/share/mamba
If your shell was already initialized, reinitialize your shell with:
    $ micromamba shell reinit --shell cmd.exe
Otherwise, this may be an issue. In the meantime you can run commands. See:
    $ micromamba run --help

Supported shells are {bash, zsh, csh, posix, xonsh, cmd.exe, powershell, fish, nu}.
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.11.40
** Copyright (c) 2021 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
'xonsh' is not recognized as an internal or external command,
operable program or batch file.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -29,6 +29,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment.yml
           create-args: >-
             python=${{ matrix.python-version }}
@@ -113,6 +114,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -233,6 +235,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -312,6 +315,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: src/ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -368,6 +372,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -418,6 +423,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -486,6 +492,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment.yml
           create-args: >-
             python=${{ matrix.python-version }}
@@ -747,6 +754,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux_llvm.yml
           create-args: >-
             llvmdev=${{ matrix.llvm-version }}
@@ -805,6 +813,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux_llvm.yml
           create-args: >-
             llvmdev=${{ matrix.llvm-version }}
@@ -901,6 +910,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             llvm-openmp
@@ -942,6 +952,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -961,6 +972,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -980,6 +992,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -1030,6 +1043,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -1072,6 +1086,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux_llvm.yml
           create-args: >-
             python=3.10
@@ -1117,6 +1132,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -1172,6 +1188,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_linux.yml
           create-args: >-
             python=3.10
@@ -1205,6 +1222,7 @@ jobs:
 
       - uses: mamba-org/setup-micromamba@v1.8.0
         with:
+          micromamba-version: '1.5.10-0'
           environment-file: ci/environment_docs_linux.yml
 
       - uses: hendrikmuhs/ccache-action@main

